release_management/_utils.py

```diff
@@ -261,6 +263,8 @@ def make_source_tarball(package_name, release_version, output_dir):
                     recursive=False,
                 )
 
+    return output_file
```

Comment:
Returning ```output_file``` is correct, but it is already a path when you return it. In fact, the way it is used now, it is already always an absolute path, because ```output_dir``` is always an absolute path when ```make_source_tarball``` is called.

So, you should do the following changes:

1. Make an assert at the top of this method, like: ```assert isabs(output_dir), f"{output_dir} is not an absolute path"```.

2. Wherever you call ```make_source_tarball``` drop the change where you make the return value an absolute path, because it already is.

3. Fix up the comments in the header method, so that is specifies that the return value is the absolute path of the source tarball.

Reply:
I think all of this is resolved; I also renamed `output_abs_path` to `output_path` as a cleanup, and since that part of the code shouldn't have to distinguish whether that variable is an absolute path or not.